bench/benchmark.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var benchmark = require('benchmark');
+var Table = require('cli-table');
+var colors = require('colors/safe');
+
+var oldZ = require('sanctuary-type-classes');
+var newZ = require('..');
+
+
+function identity(x) {
+  return x;
+}
+
+function formatPct(pct) {
+  return ('+' + pct.toLocaleString('en', {
+    minimumIntegerDigits: 3,
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+    useGrouping: false
+  })).slice(-6) + '%';
```

Comment:
That's a really ugly way to slice the `+` off the `+-` case. I should rewrite this function in a more sane way.